reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Variables.md

```diff
@@ -93,14 +93,53 @@ $Splat = @{ Name = "Win*"; Include = "WinRM" }
 Invoke-Command -Session $s -ScriptBlock { Get-Service @Using:Splat }
 ```
 
-## Using local variables in PowerShell 2.0
+### Other situations where the 'Using' scope modifier is needed
+
+For anything that executes out-of-runspace, doesn't execute directly in the
+caller's runspace, you need the `Using` scope modifier in order to embed
+variable values from the caller's scope, so that the out-of-runspace code
+can access it. (Conversely, all other contexts neither require nor support the
+'Using' scope modifier.) Specifically, this includes the following contexts:
+
+- Remotely executed commands, started with Invoke-Command's `-ComputerName` or
+  `-Session` parameter (remote runspace)
+- Background jobs, started with Start-Job (out-of-process runspace)
+- Thread jobs, started via Start-ThreadJob (separate runspace)
+- InlineScript in Workflows (out-of-process)
+
+> [!NOTE]
+> Note that out-of-runspace code can never modify variables in the caller's
+> scope. However, in the case of thread jobs (but not during remoting and not in
+> background jobs), if the variable value happens to be an instance of a reference
+> type (like a collection type), it is possible to modify that instance in
+> another thread, which requires synchronizing the modifications across threads,
+> should multiple threads perform modifications.
+
+## Serialization of variable values
+
+Remotely executed commands and background jobs run out-of-process. For values
+in variables to cross these process boundaries they undergo XML-based
+serialization and deserialization. This typically involves loss of type fidelity
+\- both on input and output. --> todo: example needed
+
+Thread jobs, by contrast, because they run in a different runspace (thread) in
```

Comment:
Should use 'session' instead of 'runspace', and 'variable reference' to be consistent.